### ISSUE TITLE
Computationally determine thermostability

### ISSUE THREAD

**Issue by QuigDevs** [petadex/root#10]
**Pixeldom4:**

> Benchmark a variety of models to computationally assay the thermostability and melting point of the protein from sequence/structure.
> 
>     TemBERTure
>     ThermoProt
>     ProtT5
>     ThermoPalm (ProtT5-XL-U50)
> 
> My main concern here is that I need to benchmark these to verify two things:
> 
>     Accuracy for the SEO dataset
>     Not phylogenetically overfitting
> 
> I don't think that these should be phylogenetically overfitting (by nature of the large datasets used to train these models, in order of tens/hundreds of thousands of proteins), however, due to how ThermoPalm is fine-tuned for PETase like proteins, it is entirely possible (model needs to be more robustly validated).
